src/test/java/com/karankumar/bookproject/backend/service/BookServiceTest.java

```diff
@@ -246,4 +253,51 @@ private Book createBookAndSetAllAttributes() {
     void deleteBooks() {
         bookService.deleteAll();
     }
+    
+    @Test
+    void bookWithoutPublisherWithAuthorSaved() {
+    	SoftAssertions softly = new SoftAssertions();
```

Comment:
In line with our style guide, assertSoftly should be used here (saves us from needing to assert all at the end)